tools/ARIAtools/extractProduct.py

```diff
@@ -294,22 +300,33 @@ def merged_productbbox(product_dict, workdir='./', bbox_file=None, croptounion=F
         # Generate footprint for the common intersection of all products
         else:
             prods_bbox=prods_bbox.intersection(total_bbox)
-        # Check if there is any common overlap
-        if prods_bbox.bounds==():
-            raise Exception('No common overlap, footprint cannot be generated. Last scene checked: %s'%(scene['productBoundingBox'][0]))
-        save_shapefile(prods_TOTbbox, prods_bbox, 'GeoJSON')
+            # Estimate percentage of overlap with bbox
+            if prods_bbox.bounds==():
+                print('Rejected scene %s has no common overlap with bbox'%(scene['productBoundingBox'][0]))
+                rejected_scenes.append(product_dict.index(scene))
+                os.remove(scene['productBoundingBox'][0])
+            else:
+                per_overlap=(prods_bbox.area/total_bbox.area)*100
```

Comment:
Originally this is an area, but gets converted to a percentage. An area threshold would be something that can be related across AOI's provided. Threshold based on percentage becomes a bit more AOI specific.

Reply:
But this percentage is then directly compared to the user-defined percentage of overlap. Are you suggesting then instead to have the user-defined input in units of m^2 or km^2?

Reply:
Im referring to a threshold that can be applied across different AOI's and have a physical meaning. The percentage requires knowledge in advance on the AOI extend. E.g. the maximum union etc could change if there is a poor product being generated.

Reply:
I agree, if this is practical. I had originally kicked out products based on fraction of the frame area in part for this reason... because the frame areas are relatively consistent, but one doesn't need to know the exact area of a frame in km2. 

But otherwise, I follow your logic that it is better to implement this criterion in the lower part of the script, after the merged area has been defined.

Reply:
Frames are only consistent within AOI, but not on AOI boudaries.

Reply:
Yeah, it seems you would need to have a rough idea of the AOI extent a priori. I've already started to have problems for instance with the CA study area because track 87 exists in both CA and Saudi Arabia.

So I'm not sure of the best way to proceed without at least specifying a rough bbox.

Currently if no bbox is specified, the code takes the longest scene and uses it as a spatial reference (i.e. its area becomes the "bbox"), see lines 289-292.